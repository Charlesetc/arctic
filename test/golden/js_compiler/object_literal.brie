define main [
  < x = 2, y = "hi" >
]
