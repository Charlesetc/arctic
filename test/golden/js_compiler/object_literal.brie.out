const global_functions = {1: {}};
function main() {
	return {x: 2, y: "hi"};
}
console.log(main());
var test/golden/js_compiler/object_literal.brie_main = new_closure(1)